igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/client/model/Config.groovy

```diff
@@ -26,18 +27,26 @@ import org.simpleframework.xml.Default
 @JsonInclude(JsonInclude.Include.NON_NULL)
 class Config {
 
-    List<String> env
+    @SerializedName("global_env")
+    List<Object> globalEnv
+
+    List<String> env = ["SPINNAKER_TRIGGERED=true"]
```

Comment:
This part is also quite strange. We need to override a random env variable in order to override the global_env. Travis is looking into this as well.